torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -746,6 +746,13 @@ RegisterOperators reg(
            return 0;
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::MakeTensor() -> Tensor",
+         [](Stack& stack) {
+           push(stack, at::Tensor());
+           return 0;
+         },
+         aliasAnalysisFromSchema()),
```

Comment:
The tests fail if there is no registered operator. I guess that sort of makes sense, but it would be awesome if there was a way to register an operator just for the sake of the unit tests.

Reply:
You can register operators in test files. https://github.com/pytorch/pytorch/blob/806e7daa1fbe42380db04f7b44697975b3dd13c3/test/cpp/jit/test_misc.cpp#L954

Reply:
Yeah, we should register whatever set of standard testing operators in a `test_base.cpp`.

Reply:
Added a `test_base.cpp` and registered the op there.